### PR TITLE
Upload source maps to Sentry from webpack

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Add sentry webpack plugin.
Remove thread loader (doesn't work with sentry webpack plugin).
Conditionally include Sentry webpack plugin if sentry auth token is defined.

## Why?

We want more informative exception logging in Sentry.

Webpack thread-loader isn't compatible with the Sentry webpack plugin.
Ref: https://github.com/getsentry/sentry-webpack-plugin/issues/272

_Referenced ticket(s): [GRW-379]_

[GRW-379]: https://hedvig.atlassian.net/browse/GRW-379

### CODE REVIEW

File: webpack/webpack.config.server.js [hunk 1]
```diff
@@ -52,5 +52,16 @@ module.exports = webpackConfig({
         {},
       ),
     }),
+    ...(process.env.SENTRY_AUTH_TOKEN
+      ? [
+          new SentryWebpackPlugin({
+            release: process.env.HEROKU_SLUG_COMMIT,
```
Comment (styrelius) [inline webpack/webpack.config.server.js:58]: Is this env variable `HEROKU_SLUG_COMMIT` needed to test the integration locally? Do we need to able to test it locally after we've merged? And if so, should `HEROKU_SLUG_COMMIT` be added to `.env.example`? 🤔 

And maybe the same questions goes for the `SENTRY_AUTH_TOKEN` env variable?

**Review by styrelius — COMMENTED** [review]
Nice! 🙌 Just some questions about the env variables

File: webpack/webpack.config.server.js (hunk 1 above)
Comment (robinandeer) [reply webpack/webpack.config.server.js:58]: Ahh yeah I could add those there and comment that they are optional :-)

Comment (styrelius) [reply webpack/webpack.config.server.js:58]: Awesome!! 🙌 😀

**Review by styrelius — DISMISSED** [review]
🗺️ 🤸‍♀️ 🤸‍♀️ 🤸‍♀️

File: .env.example [hunk 2]
```diff
@@ -12,3 +12,7 @@ ADYEN_ORIGIN_KEY=pub.v2.something-kinda-secret
 ADYEN_ENVIRONMENT=<test|live>
 
 APP_ENVIRONMENT=<development|staging|production>
+
+# Optional - if you want to test uploading source maps to Sentry
+HEROKU_SLUG_COMMIT=<commit hash>
+SENTRY_AUTH_TOKEN=<sentry auth token>
```
Comment (styrelius) [inline .env.example:18]: Maybe the `SENTRY_AUTH_TOKEN` should also be added to this file?

Comment (robinandeer) [reply .env.example:18]: What do you mean? It's in the file, no?

Comment (styrelius) [reply .env.example:18]: Hahaha what?? I'm blind apparently? That's kind of scary actually

**Comment by robinandeer** [comment]
Just had ro rebase to solve the conflict in `yarn.lock`

**Review by styrelius — APPROVED** [review]

**Pull request merged**